### PR TITLE
Remove gatekeeper screen

### PR DESCRIPTION
**Pull request by rudigiesler**
As part of the UET feedback, we want to remove the gatekeeper selection screen, and assume that the mother is the recipient for the messages.

### CODE REVIEW

**Comment by rudigiesler** [comment]
@KaitCrawford Ready for review.

File: test/ussd_health_worker.test.js [hunk 1]
```diff
@@ -136,34 +136,11 @@ describe("familyconnect health worker app", function() {
...
-                    })
-                    .run();
-            });
             it("to state_msisdn", function() {
                 return tester
                     .setup.user.addr('0820000111')
                     .inputs(
                         {session_event: 'new'}  // dial in
-                        , '12345'  // state_auth_code - personnel code
-                        , '1'  // state_msg_receiver - head of household
```
Comment (KaitCrawford) [inline test/ussd_health_worker.test.js:166]: why can the personnel code above be removed?

**Review by KaitCrawford — COMMENTED** [review]
@rudigiesler just one question from me

File: test/ussd_health_worker.test.js (hunk 1 above)
Comment (rudigiesler) [reply test/ussd_health_worker.test.js:166]: I think this was actually an error in the test. The number above is a recognised number, so it shouldn't ask for a personnel code. I think what was happening is that it was asking for the message receiver, `12345` was put in as the input, which was not a valid input, so the head of household error screen was presented, where they put in `1`, which is a valid input, and so it went onto the next state as normal.

Comment (KaitCrawford) [reply test/ussd_health_worker.test.js:166]: Ah ok

**Comment by KaitCrawford** [comment]
:+1:

**Pull request merged**